chat-microservice/app/api/v1/chat.py

```diff
@@ -61,3 +62,31 @@ async def get_user_chats(current_user_id = Depends(get_current_user_id)):
     """
     results = Chat.objects().filter(users_id__contains=str(current_user_id)).allow_filtering()
     return list(results)
+
+@router.get(
+    "/chat/messages/", 
+    status_code=status.HTTP_200_OK,
+)
+async def get_messages(chat_id: str, time: str = None, quantity: int = 5):
+    """
+    Get chat messages between users using this endpoint.
+    - **chat_id**: ID of the chat between users
+    - **time**: get the chat messages sorted by the time
+    - **quantity**: limit of the messages that you will receive
+    """
+    list_msg = []
+    query = ChatMessages.objects(chat_id=chat_id).order_by("time")
+    if time == None:
+        time = str(query.first()["time"])
+    time_get = False
+    index = 0
+    for msg in query:
+        if str(msg.time) == time or time_get and index < quantity:
+            list_msg.append(msg)
+            time_get = True
+            index = index+1
+    if not list_msg:
+        raise HTTPException(
+            status_code=422, detail='there is no chat messages corresponding to those specifications'
```

Comment:
We need messages and validations for the next case:

There are not pages for the given time 404
The chat_id is not found 404
Max quantity allowed should be 20 validate if the query params is greater or lower than 0 . I think you can use pydantic for this.

Reply:
All the messages strings should be in the config file like `settings.CASSANDRA_MESSAGE_CREATION_UNAUTHORIZED`

Reply:
save 20 AS constant in config.py something like :MAX_MESSAGES_QUntity_pagination